.gitignore

```diff
@@ -224,3 +224,4 @@ ui/.vercel
 
 # Go subprocess binaries (built during feast pip package building)
 sdk/python/feast/binaries/
+=4.0.30
```

Comment:
What is this for?

Reply:
Not sure why this strange file popped up in my folders, was installing a package in a different window of vscode of this version, and this file started to show up here. 

This file is unrelated and not required for this PR, and hence in gitignore.

Reply:
I think you have a bad file in your local workspace; can you delete it from there instead of adding it to the project's .gitignore?